verifyFiles.py

```diff
@@ -253,6 +253,7 @@ def main():
                                              /home/$(USER)/CentOS6.FSL5.0.6
                                              /home/$(USER)/CentOS7.FSL5.0.6
                                              Each directory will contain subject folders like 100307,100308 etc'''))
+        parser.add_argument("-c", "--checksumfile",action="store_true",help="Reads checksum from files. Doesn't compute checksums locally") 
```

Comment:
It doesn't look like this option was implemented in the code yet. In such cases, it's better to wait for the whole thing to be available before pushing to a pull request.